Bottom: c77e15879e929a7bcd9545d9e332138caa455fcb
Top:    30a9e4c23aefe1df47a45261cd1cbcb57e198c3b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-15 18:39:54 +0000

find.c: Remove unused function lineClear()


---

diff --git a/src/find.c b/src/find.c
index d03bc8c..b65fc51 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2805,21 +2805,6 @@ DoIt (bool AndRats, bool AndDraw)
   return (newone);
 }
 
-/* returns true if nothing un-found touches the passed line
- * returns false if it would touch something not yet found
- * doesn't include rat-lines in the search
- */
-
-bool
-lineClear (LineType *line, Cardinal group)
-{
-  if (LOTouchesLine (line, group))
-    return (false);
-  if (PVTouchesLine (line))
-    return (false);
-  return (true);
-}
-
 /* ---------------------------------------------------------------------------
  * prints all unused pins of an element to file FP
  */
diff --git a/src/find.h b/src/find.h
index b74a668..d811634 100644
--- a/src/find.h
+++ b/src/find.h
@@ -66,7 +66,6 @@ void RatFindHook (int, void *, void *, void *, bool, bool);
 void SaveFindFlag (int);
 void RestoreFindFlag (void);
 int DRCAll (void);
-bool lineClear (LineType *, Cardinal);
 bool IsLineInPolygon (LineType *, PolygonType *);
 bool IsArcInPolygon (ArcType *, PolygonType *);
 bool IsPadInPolygon (PadType *, PolygonType *);
